### DISCUSSION TITLE
Starting Subfigure Captions with Parenthesis

### DISCUSSION THREAD

**Discussion by betanalpha** [quarto-dev/quarto-cli#9132] [Q&A] [answered]
### Description

Quarto Version: 1.4.550
OS: macOS 14.2

Occasionally I want to begin a subfigure caption with a parenthesis, for example with using a pattern like "(a) Description of first subfigure, (b) description of second subfigure."  Unfortunately in `quarto` this renders with the entire caption located under the first subfigure and no global caption.

As an example the notebook

````qmd
---
title: "Test"
format:
  html:
    theme:
      - lux
---

::: {#fig-label layout="[ [50, 50] ]"}

![](test){#fig-label-sub1}

![](test){#fig-label-sub2}

This caption references (a) the first subfigure and (b) the second 
subfigure.
:::

::: {#fig-label layout="[ [50, 50] ]"}

![](test){#fig-label-sub1}

![](test){#fig-label-sub2}

(a) The first subfigure and (b) the second subfigure.
:::
````

renders as

<img width="727" alt="html_screenshot" src="https://github.com/quarto-dev/quarto-cli/assets/1527190/a68edf88-fae8-42aa-b8b6-3288f53a59cb">

Is there anyway to escape the subfigure caption so that the text "(a) The first subfigure and (b) the second subfigure." is treated as a global caption and renders after "Figure 2"?

Relatedly is there anyway to line break the `![](filename){#subfig-label}` pattern without messing up the subfigure rendering?  I often have long file names and/or long figure labels but if I try to line break to, for example,
```
![](filename)
{#subfig-lable}
```
then the subfigure doesn't render correctly.

Thanks!

**Comment by cscheid** [comment]
Let's see if we can improve it bit by bit.

I'd start with:

<table><tr><th>qmd</th><th>html</th></tr><tr><td>

````
::: {#fig-label layout="[ [50, 50] ]"}

![the first subfigure](https://placeholder.co/400/400){#fig-label-sub1}

![the second subfigure](https://placeholder.co/400/400){#fig-label-sub2}
:::
````

</td><td>

<img width="851" alt="image" src="https://github.com/quarto-dev/quarto-cli/assets/285675/7651ba37-ddef-4837-b81b-881a409474b7">

</td></tr></table>

That overall caption with "Figure 2" that we produce is not ideal, but maybe that's acceptable for your use case. If not, then the easiest fix is for you to tweak your CSS, like so:

<table><tr><th>code</th><th>html</th></tr><tr><td>

Front matter:

```yaml
title: "Test"
format:
  html:
    css:
      - uncaptioned_hack.css
    theme:
      - lux
```

uncaptioned_hack.css:

```css
figcaption.quarto-uncaptioned {
    display: none;
}
```
</td><td>

<img width="836" alt="image" src="https://github.com/quarto-dev/quarto-cli/assets/285675/cb140ef9-c336-4443-bef8-8d71d52869ea">

</td></tr></table>

Here we're taking advantage of the fact that in HTML, Quarto tags figcaption elements with no captions with the `quarto-uncaptioned` class.

**Reply by betanalpha** [reply]
Apologies for the ambiguity: I'm interested in the opposite behavior.

I do not want individual subfigure captions but rather a single global caption that would render as "Figure 2: (a) The first subfigure and (b) the second subfigure.", notice the parenthesis immediate after the "Figure 2:", with no text after the automatically generated `(a)` and `(b)` subfigure labels.

In other words I'm looking for a way to escape the global caption text to avoid automatically moving it to subfigure captions.

I can try to edit a photo of the desired render behavior if that would help.

**Comment by cscheid** [comment]
Re: your figure-splitting question, there's nothing much we can do here because the specific parsing is coming from Pandoc, but the following might be an acceptable workaround:

```
![](filename){
#subfig-lable}
```

**Comment by cscheid** [comment, marked as answer]
Another markdown comment: starting a paragraph with parentheses causes Pandoc to think you're starting a list:

```
$ pandoc -f markdown -t native
(a) one element
(b) another element
^D
[ OrderedList
    ( 1 , LowerAlpha , TwoParens )
    [ [ Plain [ Str "an" , Space , Str "item" ] ]
    , [ Plain [ Str "another" , Space , Str "item" ] ]
    ]
]
```

So that's why you're seeing that weirdness in your first attempt at a caption. To start a paragraph with parentheses in Pandoc, use `\(a)` (I know, it's weird).

**Reply by betanalpha** [reply]
Ah that's it, thanks!  

````qmd
::: {#fig-label layout="[ [50, 50] ]"}

![](test){#fig-label-sub1}

![](test){#fig-label-sub2}

\(a) The first subfigure and (b) the second subfigure.
:::
````

renders as

<img width="846" alt="screenshot" src="https://github.com/quarto-dev/quarto-cli/assets/1527190/07ebaa56-3d98-4bd3-ba89-ac993ac6ffbb">

as desired.

I swear I tried a backslash escape previously but I clearly hadn't.  Thanks, again!
